resources/core/charts/console/templates/kubernetes-dashboard-admin.yaml

```diff
@@ -1,14 +1,38 @@
+# apiVersion: rbac.authorization.k8s.io/v1beta1
+# kind: ClusterRoleBinding
+# metadata:
+#   name: kubernetes-dashboard-kyma-admin
+#   labels:
+#     app: kyma
+# subjects:
+# - kind: ServiceAccount
+#   name: kubernetes-dashboard
+#   namespace: kube-system
+# roleRef:
+#   kind: ClusterRole
+#   apiGroup: rbac.authorization.k8s.io
+#   name: kyma-admin
+
+
 apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: ClusterRoleBinding
 metadata:
-  name: kubernetes-dashboard-kyma-admin
+  name: kubernetes-dashboard
   labels:
-    app: kyma
+    k8s-app: kubernetes-dashboard
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
 subjects:
 - kind: ServiceAccount
   name: kubernetes-dashboard
   namespace: kube-system
-roleRef:
-  kind: ClusterRole
+- kind: User
+  name: admin@kyma.cx
   apiGroup: rbac.authorization.k8s.io
-  name: kyma-admin
\ No newline at end of file
+- kind: ServiceAccount
+  name: kube-rbac-proxy  
+  namespace: kube-system
+- kind: Group
```

Comment:
group "/admin" ? did you mean "kyma-admins" ?

Reply:
This was a temp. permissions file for tests. Def. will be reverted 👍